src/javascript/core/transports/image.js

```diff
@@ -20,3 +20,6 @@ module.exports = function () {
 		}
 	};
 };
+
+export default image;
+export { image };
```

Comment:
this is tricky, these are different between commonjs, commonjs w/ interop and es6

not sure what the best option is here

Reply:
here, the codeblocks contain the value of the import.

i might be wrong about this, but i think this results in the following:

### commonjs without noInterop `let image = require("./image")`

```js
{image: image, default: image}
```

### commonjs with noInterop `let image = require("./image")`

```js
{default: {image: image, default: image}}
```

### es6 `import image from "./image"`
```
image
```

### es6 `import {image} from "./image"`
```
image
```

Reply:
i may well be wrong and someone sould probably try it out